libs/velo-external-db-commons/src/libs/errors.ts

```diff
@@ -1,90 +1,106 @@
 class BaseHttpError extends Error {
-    status: number
-    constructor(message: string, status: number) {
+    constructor(message: string) {
         super(message)
-        this.status = status
     }
 }
 
 export class UnauthorizedError extends BaseHttpError {
     constructor(message: string) {
-        super(message, 401)
+        super(message)
```

Comment:
Why did you remove all the status codes?

Reply:
Since they are now irrelevant. The status code is determined via errors.ts